bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -213,7 +223,7 @@ protected static EventLoopGroup createDefaultEventLoopGroup() {
             .build());
   }
 
-  protected static ScheduledExecutorService createDefaultRetryExecutor() {
+  protected ScheduledExecutorService createDefaultRetryExecutor() {
```

Comment:
but I thought we are letting user pass in ScheduledExecutorService. I prefer not allowing user to override this, up to you though.